src/core/GOMemoryPool.cpp

```diff
@@ -28,39 +28,33 @@
 #endif
 #include <errno.h>
 
-static inline void touchMemory(const char* pos) { load_once(*pos); }
+static inline void touchMemory(const char *pos) { load_once(*pos); }
 
 GOMemoryPool::GOMemoryPool()
-    : m_PoolStart(0),
-      m_PoolPtr(0),
-      m_PoolEnd(0),
-      m_CacheStart(0),
-      m_PoolSize(0),
-      m_PoolLimit(0),
-      m_PageSize(4096),
-      m_CacheSize(0),
-      m_MallocSize(0),
-      m_MemoryLimit(0),
-      m_AllocError(0),
-      m_TouchPos(0),
+    : m_PoolStart(0), m_PoolPtr(0), m_PoolEnd(0), m_CacheStart(0),
+      m_PoolSize(0), m_PoolLimit(0), m_PageSize(4096), m_CacheSize(0),
+      m_MallocSize(0), m_MemoryLimit(0), m_AllocError(0), m_TouchPos(0),
```

Comment:
It is not desired.

Please add

```
ConstructorInitializerIndentWidth: 2
ConstructorInitializerAllOnOneLineOrOnePerLine: true
```

to the `.clang-format`

Reply:
> ConstructorInitializerAllOnOneLineOrOnePerLine: true

That one is already in place

Reply:
> ConstructorInitializerIndentWidth: 2

Done

Reply:
> > ConstructorInitializerAllOnOneLineOrOnePerLine: true
> 
> That one is already in place

No, it isn't. It should place every `m_PoolSize(0),` in a separate line

Reply:
It did it:

![image](https://user-images.githubusercontent.com/19529533/148655885-27094e92-241a-4c59-8219-e4402f6a2895.png)


You need to go to files in Github.

Reply:
![image](https://user-images.githubusercontent.com/19529533/148655933-146253e3-5564-4e14-ba20-cc97e32b2cbb.png)